examples/plot/app.js

```diff
@@ -46,17 +46,31 @@ class Root extends Component {
     });
   }
 
+  _onHover(info) {
+    const hoverInfo = info.sample ? info : null;
+    if (hoverInfo !== this.state.hoverInfo) {
+      this.setState({hoverInfo});
+    }
+  }
+
   render() {
-    const {viewport} = this.state;
+    const {viewport, hoverInfo} = this.state;
 
     return (
       <OrbitController
         {...viewport}
         onChangeViewport={this._onChangeViewport.bind(this)} >
-        <DeckGLOverlay viewport={viewport}
+        <DeckGLOverlay
+          viewport={viewport}
           equation={equation}
           resolution={200}
-          showAxis={true} />
+          showAxis={true}
+          onHover={this._onHover.bind(this)} />
```

Comment:
Better to pre-bind this in constructor as `Function.bind()` generates a new function every render and can thus defeat `shouldComponentUpdate`.